odl/discr/lp_discr.py

```diff
@@ -404,8 +373,7 @@ def _inner(self, x, y):
             return super()._inner(x, y)
         else:
             # TODO: implement without copying x
-            func_list = _scaling_func_list(bdry_fracs)
-
+            func_list = _scaling_func_list(bdry_fracs, exponent=1.0)
```

Comment:
Because exponent is only used for norm if something to the p-th power should be integrated. It's added here because it was an optional parameter in the helper before, which didn't really make sense.